content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -156,52 +156,51 @@ <h2 id="kbd_label">Keyboard Support</h2>
 
       <section>
-        <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+        <h2 id="rps_label">ロール、プロパティ、ステート、及び tabindex 属性</h2>
         <table aria-labelledby="rps_label" class="data attributes">
           <thead>
             <tr>
-              <th scope="col">Role</th>
-              <th scope="col">Attributes</th>
-              <th scope="col">Element</th>
-              <th scope="col">Usage</th>
+              <th scope="col">ロール</th>
+              <th scope="col">属性</th>
+              <th scope="col">要素</th>
+              <th scope="col">使用法</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="radiogroup-role">
               <th scope="row"><code>radiogroup</code></th>
               <td></td>
               <td><code>ul</code></td>
-              <td>Identifies the element as a container for a group of <code>radio</code> buttons.</td>
+              <td>要素を <code>radio</code> ボタンのグループのコンテナとして識別します。</td>
             </tr>
             <tr data-test-id="radiogroup-aria-labelledby">
               <td></td>
               <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
               <td><code>ul</code></td>
-              <td>Refers to the element that contains the label of the radio group.</td>
+              <td>ラジオグループのラベルを含む要素を参照します。</td>
             </tr>
             <tr data-test-id="radiogroup-tabindex">
               <td></td>
-              <th scope="row"><code>tabindex=&quot;0&quot;</code></th>
+              <th scope="row"><code>tabindex="0"</code></th>
               <td><code>ul</code></td>
               <td>
                 <ul>
-                  <li>Includes the radio group in the page <kbd>Tab</kbd> sequence.</li>
-                  <li>Applied to the radio group because <code>aria--activedescendant</code> is used to manage focus as described below.</li>
+                  <li>ラジオグループをページの <kbd>Tab</kbd> 順序に含めます。</li>
+                  <li>以下で説明するように、フォーカスを管理するために <code>aria--activedescendant</code> が使用されるため、ラジオグループに適用されます。</li>
                 </ul>
               </td>
             </tr>
             <tr data-test-id="radiogroup-aria-activedescendant">
               <td></td>
-              <th scope="row"><code>aria-activedescendant=&quot;[IDREF]&quot;</code></th>
+              <th scope="row"><code>aria-activedescendant="[IDREF]"</code></th>
               <td><code>ul</code></td>
               <td>
                 <ul>
-                  <li>When a radio button in the radio group is visually indicated as having keyboard focus, refers to that radio button.</li>
-                  <li>When arrow keys are pressed, the JavaScript changes the value.</li>
-                  <li>Enables assistive technologies to know which element the application regards as focused while DOM focus remains on the radio group element.</li>
+                  <li>ラジオグループ内のラジオボタンが視覚的にキーボードフォーカスを持っているとして示されている場合、そのラジオボタンを参照します。</li>
```

Comment:
修飾を明確にするために語順を変更した方がよさそうです
```suggestion
                  <li>ラジオグループ内のラジオボタンがキーボードフォーカスを持っていると視覚的に示されている場合、そのラジオボタンを参照します。</li>
```